### PR TITLE
WEB-100: Support for setting an API key

### PR DESCRIPTION
**Pull request by rmil**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,48 @@
...
-The build is minified and the filenames include the hashes.<br />
 Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
-### `yarn eject`
+## Basic breakdown of how to edit this:
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+index.tsx to change theme
```
Comment (cs-endo) [inline README.md:40]: Put backticks around this filename

File: README.md [hunk 2]
```diff
@@ -1,47 +1,48 @@
...
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+App.tsx to add pages or change page routing in react Router
```
Comment (cs-endo) [inline README.md:42]: Put backticks around this filename

File: README.md [hunk 3]
```diff
@@ -1,47 +1,48 @@
...
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+NavbarWithDrawer.tsx to change/add menu links/icons
```
Comment (cs-endo) [inline README.md:44]: Put backticks around this filename

**Review by cs-endo — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (rmil) [reply README.md]: Addressed in 1204714

File: README.md (hunk 2 above)
Comment (rmil) [reply README.md]: Addressed in 1204714

File: README.md (hunk 1 above)
Comment (rmil) [reply README.md]: Addressed in 1204714

**Pull request merged**